### DISCUSSION TITLE
並び替え→並べ替え（sortingの訳語）

### DISCUSSION THREAD

**Discussion by smatsudajp** [pgsql-jp/jpug-doc#2716] [NGとする語]
## 問題点

sortingの訳語として「並び替え」と「並べ替え」が使われているが、「並び替え」は約40箇所、「並べ替え」は約10箇所と、「並び替え」が圧倒的に多い。
対訳リストでもsortingの訳語は「並び替え」となっている。
しかし、日本語文法としては「並べ替え」の方が正しいと考えられる。
## 背景

例えば、自動詞の「付く」、他動詞の「付ける」に対して、「付け替え」はあっても「付き替え」はないことから類推すると、自動詞の「並ぶ」、他動詞の「並べる」に対して「並べ替え」が正しいと考えるのが自然。
また「並び替え」でGoogle検索すると、上位に出てくるのはすべて「並べ替え」である。
## 解決方法

適当なタイミングで一括置換。
## 注意点

直感的には一括置換でよさそうだが、「並び替え」の方が適切な場合というのは本当に存在しないのか。一括置換ではなく、個別に確認しながら置換すべきか？

**Comment by koizumistr** [comment]
確かに MS Office や LibreOffice でも「並べ替え」になってますね。

**Comment by harukat** [comment]
（6/15 翻訳者会議）
「並べ替え」に寄せる。クローズは既存訳変更後。
どちらかといえば「並び替え」が良い場合があるかもしれない件は、将来の指摘を待つことで対処。

**Comment by koizumistr** [comment]
#2712 にも書きましたが、本件、「並び替え」という名詞が対象であって、「並び替える」という動詞は対象ではないと勝手に思っていたのですが、動詞を対象に含めても

> どちらかといえば「並び替え（る）」が良い場合があるかもしれない

というのはありませんでした。（個人的には、名詞よりも動詞の方が許せるかな、くらいです。が、「並べ替える」の方がいいかな、というのは変わりません。）

**Comment by noborus** [comment]
漢字違いもあったので以下もNG語にする

並び換え
並び変え
並べ換え
並べ変え
